internal/host/statemachine.go

```diff
@@ -611,20 +611,6 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th TransitionHandler) stat
 		},
 	})
 
-	sm.AddTransitionRule(stateswitch.TransitionRule{
-		TransitionType: TransitionTypeRefresh,
-		SourceStates: []stateswitch.State{
-			stateswitch.State(models.HostStatusInstalling),
-		},
-		Condition:        stateswitch.Not(If(IsConnected)),
-		DestinationState: stateswitch.State(models.HostStatusError),
-		PostTransition:   th.PostRefreshHost(statusInfoConnectionTimedOut),
-		Documentation: stateswitch.TransitionRuleDoc{
-			Name:        "Move installing host to error when connection times out",
-			Description: "TODO: Document this transition rule.",
-		},
-	})
-
```

Comment:
Which transition covers this one now?

Reply:
There is basically another transition with the same state in an or condition. So there was no need for this one.

@ Line 629
`
	sm.AddTransitionRule(stateswitch.TransitionRule{
		TransitionType: TransitionTypeRefresh,
		SourceStates: []stateswitch.State{
			stateswitch.State(models.HostStatusInstalling),
			stateswitch.State(models.HostStatusInstallingInProgress),
		},
		Condition:        stateswitch.Not(If(IsConnected)),
		DestinationState: stateswitch.State(models.HostStatusError),
		PostTransition:   th.PostRefreshHost(statusInfoConnectionTimedOutInstalling),
		Documentation: stateswitch.TransitionRuleDoc{
			Name:        "Move installing host to error when connection times out",
			Description: "TODO: Document this transition rule.",
		},
	})`